extensions/GitExtension/FileExplorerGitIntegration.UnitTest/GitLocalRepositoryProviderUnitTests.cs

```diff
@@ -100,6 +100,6 @@ public void GetProperties()
         Assert.AreEqual(result["System.VersionControl.LastChangeAuthorEmail"], "a.u.thor@example.com");
         Assert.AreEqual(result["System.VersionControl.LastChangeAuthorName"], "A U Thor");
         Assert.AreEqual(result["System.VersionControl.LastChangeID"], "d0114ab8ac326bab30e3a657a0397578c5a1af88");
-        Assert.AreEqual(result["System.VersionControl.CurrentFolderStatus"], "master AheadBy: 0 BehindBy: 0");
+        Assert.AreEqual(result["System.VersionControl.CurrentFolderStatus"], "Branch: master ≡ | +0 ~0 -0 | +0 ~0 -0");

```

Comment:
I thought branches use `main` instead of `master`

Reply:
> I thought repositories use `main` instead of `master`

Fun(?) fact: git still defaults to an initial branch of `master`, but this is configurable. See https://git-scm.com/docs/git-init

GitHub.com *now* defaults to `main` when it creates a repo for you behind the scenes.

This test is using an ancient test repository at https://github.com/libgit2/TestGitRepository, whose default branch is `master`.